src/VisualStudio/Core/Test/DebuggerIntelliSense/TestState.vb

```diff
@@ -76,9 +65,8 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.DebuggerIntelliSense
 
             Me.SessionTestState = GetExportedValue(Of IIntelliSenseTestState)()
 
-            Me.AsyncCompletionService = Workspace.GetService(Of IAsyncCompletionService)
-
-            Me.CompletionCommandHandler = Workspace.GetService(Of CompletionCommandHandler)
+            Me.CompletionCommandHandler = GetExportedValues(Of VSCommanding.ICommandHandler)().
+                Single(Function(e As VSCommanding.ICommandHandler) e.GetType().FullName = "Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.CompletionCommandHandler")
```

Comment:
We should be able to grab the command handler via the name so we don't have to hard-code internal types.

Reply:
We do not have a reference to the Editor implementation.

---
In reply to: [322476631](https://github.com/dotnet/roslyn/pull/38569#discussion_r322476631) [](ancestors = 322476631)